Bitly/Bitly_List_Active_Links.ipynb

```diff
@@ -0,0 +1,261 @@
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6cd85c67-0488-45bb-b50c-70018ad3590c",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "912afa92-2391-44be-be30-d3d52262ed5a",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7dd928ae-e73d-4031-adc6-c1d0bdf51dff",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup Variables\n",
+    "- **token**: [Generate a Bitly Access Token](https://dev.bitly.com/v4/#section/Authentication)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45dfd5f9-de84-4d78-be84-58639a9e45d9",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "token = \"<YOUR_TOKEN_HERE>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bfdfbe5e-2d07-42c4-bdca-ee55b432f069",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f01ad839-7e6d-4230-b72e-9308be5d1b5d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### List active links"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48041331-fc52-4b0f-b130-197a268be4f0",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "This function will list all active links associated with the Bitly account."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70525d70-aa86-425b-b41d-9d495fa44c17",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def list_active_links(token):\n",
+    "    headers = {\"Authorization\": \"Bearer \" + token}\n",
+    "    params = {\"size\": 100}\n",
+    "    response = requests.get(\n",
+    "        \"https://api-ssl.bitly.com/v4/user/active_links\", headers=headers, params=params\n",
+    "    )\n",
+    "    response.raise_for_status()\n",
+    "    return response.json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d5e7a4-afcf-447d-89d0-388b76b45cec",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Output"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49d0541a-94fb-497c-8893-4f2004a2549d",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Display result"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46436a7b-1379-4e7e-a0b4-967126e7bcbd",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "active_links = list_active_links(token)\n",
```

Comment:
<img width="1799" alt="Screenshot 2023-02-21 at 1 41 29 PM" src="https://user-images.githubusercontent.com/81156510/220286080-3c966ef4-3b99-4b2e-958b-1bada1cea715.png">

Getting `HTTPError: 404 Client Error: Not Found for url: https://api-ssl.bitly.com/v4/user/active_links?size=100` while running the notebook.